tenseal/sealapi/sealapi_evaluator.cpp

```diff
@@ -0,0 +1,302 @@
+#include <pybind11/complex.h>
+#include <pybind11/iostream.h>
+#include <pybind11/operators.h>
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+#include <seal/seal.h>
+
+#include <fstream>
+
+#include "bindings.h"
+
+using namespace seal;
+using namespace std;
+namespace py = pybind11;
+
+void bind_evaluator(pybind11::module &m) {
+    /*******************
+     * "seal/evaluator.h" {
+     ***/
+    py::class_<Evaluator, std::shared_ptr<Evaluator>>(m, "Evaluator",
+                                                      py::module_local())
+        .def(py::init<std::shared_ptr<SEALContext>>())
+        .def("add_inplace", &Evaluator::add_inplace)
+        .def("sub_inplace", &Evaluator::sub_inplace)
+        .def("multiply_inplace",
+             [](Evaluator &e, Ciphertext &l, const Ciphertext &r) {
+                 return e.multiply_inplace(l, r);
+             })
```

Comment:
Q: Is this and similar inplace functions not doing it really inplace?

Reply:
return does nothing here.
I will update these bindings with the py::arg() format.

Reply:
I removed the useless return keywords.